### PR TITLE
⚡️ add paramiko lab 2 to config router R1-R3

### PR DESCRIPTION
**Pull request by tigerza117**
Add new lab

- [x] Ubuntu can ping outsite
- [x] Ubuntu to outsite with hop in control-data
- [x] Disable control data telnet/ssh to R1-R3
- [x] PAT

### CODE REVIEW

File: paramikolab02.py [hunk 1]
```diff
@@ -4,25 +4,108 @@
...
+        "default-information originate",
+        "access-list 1 permit 172.31.111.16 0.0.0.15",
+        "access-list 1 permit 172.31.111.32 0.0.0.15",
+        "access-list 1 permit 172.31.111.48 0.0.0.15",
+        "ip nat inside source list 1 int g0/2 vrf control-data overload",
+    ])
+]
+
+for device in devices:
+    device.exec()
```
Comment (tigerza117) [inline paramikolab02.py]: Too slow when running 1 by 1

**Review by tigerza117 — COMMENTED** [review]
Ok faster now

**Comment by tigerza117** [comment]
@SoNicBoom24 What do you think.

**Review by SoNicBoom24 — APPROVED** [review]
Nice

**Pull request merged**